addons/account/models/ir_attachment.py

```diff
@@ -34,10 +34,12 @@ def _decode_edi_xml(self, filename, content):
         """Decodes an xml into a list of one dictionary representing an attachment.
         :returns:           A list with a dictionary.
         """
+
+        parser = etree.XMLParser(resolve_entities=False)
```

Comment:
This is a work in progress.

This was a problem for Italian clients, as the XMLs are also imported on third party modules' tests that have no internet connections, and the entities sometimes are not local and require the Internet to be resolved.